requirements/test.txt

```diff
@@ -7,3 +7,4 @@ pytest>=5.2.0
 pytest-cov>=2.7.0
 pytest-localserver
 pytest-faulthandler
+pytest-timeout
```

Comment:
I wonder if running the tests with something like `pytest --timeout=10` for the aarch64 wheel tests would allow those to complete. Timeouts will still get marked as failures, though, but it just wouldn't allow things to get stuck indefinitely?

Reply:
they do get marked as failures. Isn't that a major issue though? 😂

Reply:
Yes it is a major issue and it's hitting us already. Maybe that test should get skipped everywhere except, say, minreq? Or Linux 3.10? What's the fastest platform?